index.html

```diff
@@ -78,9 +83,9 @@ <h2>Why?</h2>
       </section>
 
 
-      <section>
+      <section id="team">
         <h2>Meet the team</h2>
-        <ul>
+        <ul class="flexarea">
```

Comment:
I noticed that the images are no longer centered if you shrink the page. You might be able to fix this by having them in a flex but taking up the entire row instead of one column in the middle of the row. You could still center them with flex and that way they would stay centered even as the page grows and shrinks.